examples/stub.php

```diff
@@ -691,4 +691,6 @@ public function close(){}
     public function getLastError(bool $clear = false){}
 
     public function clearError(){}
+
+    public static function strerror(int $error):?string {}
```

Comment:
This formatting is not consistent with elsewhere in the stub (`strerror(int $error) : ?string{}`)

Reply:
Fixed, do you see any other issue?